src/parquet/reader.cc

```diff
@@ -106,9 +106,9 @@ ColumnReader* RowGroupReader::Column(size_t i) {
   source->Seek(col_start);
 
   // TODO(wesm): Law of demeter violation
-  size_t bytes_read = source->Read(input->size(), input->data());
-
-  if (bytes_read != input->size()) {
+  ScopedInMemoryInputStream* pInput = static_cast<ScopedInMemoryInputStream*>(input.get());
+  size_t bytes_read = source->Read(pInput->size(), pInput->data());
```

Comment:
Google C++ forbids camel case and Hungarian notation

- https://google.github.io/styleguide/cppguide.html#Variable_Names
- https://google.github.io/styleguide/cppguide.html#Windows_Code